src/components/NavBar/ChainSwitcher.css.ts

```diff
@@ -0,0 +1,49 @@
+import { style } from '@vanilla-extract/css'
+
+import { sprinkles } from '../../nft/css/sprinkles.css'
+
+export const ChainSwitcher = style([
+  sprinkles({
+    background: 'lightGrayContainer',
+    borderRadius: '8',
+    paddingY: '8',
+    paddingX: '12',
+    cursor: 'pointer',
+    border: 'none',
+  }),
+])
+
+export const ChainSwitcherRow = style([
+  sprinkles({
+    border: 'none',
+    color: 'blackBlue',
+    justifyContent: 'space-between',
+    paddingX: '16',
+    paddingY: '12',
+    cursor: 'pointer',
+  }),
+  {
+    lineHeight: '24px',
+    width: '308px',
```

Comment:
think we want the width to be specified by the chain switcher container, then this would be width 100%

Reply:
this is the container for the dropdown which has a different width than ChainSwitcher modal in the navbar